src/OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+* `TelemetryHttpModule` will now pass the `url.path` tag (set to
+  [Request.Unvalidated.Path](https://learn.microsoft.com/dotnet/api/system.web.unvalidatedrequestvalues.path))
+  when starting `Activity` instances for incoming requests so that it is
+  available to custom samplers and may be used to influence the sampling
```

Comment:
```suggestion
  available to samplers and may be used to influence the sampling by authoring custom samplers
```

Reply:
Suggetsion: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Instrumentation.AspNet#trace-filter can mention that it is recommended to do sampling wherever feasible, and state that url.path is part of attributes available for samplers.

Reply:
@cijothomas I pushed some changes. LMK what you think.